builder/cloudbuild.yaml

```diff
@@ -0,0 +1,10 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  id: 'build'
+  script: |
+    docker build -t $LOCATION-docker.pkg.dev/$PROJECT_ID/$_CONTAINER_REPOSITORY/github-actions-runner:latest builder/.
+  automapSubstitutions: true
+images:
+- '$LOCATION-docker.pkg.dev/$PROJECT_ID/$_CONTAINER_REPOSITORY/github-actions-runner:latest'
+options:
+  logging: CLOUD_LOGGING_ONLY
```

Comment:
Please quote your yaml strings because yaml [is terrible](https://ruudvanasseldonk.com/2023/01/11/the-yaml-document-from-hell)